framework/physics/physics_material/mgxs/mgxs_read_opensn.cc

```diff
@@ -798,15 +516,15 @@ SingleStateMGXS::MakeFromOpenSnXSFile(const std::string& file_name)
 
     catch (const std::runtime_error& err)
     {
-      throw std::runtime_error("Error reading OpenSn cross section file "
+      throw std::runtime_error("Error reading Chi cross section file "
                                "\"" +
                                file_name + "\".\n" + "Line number " + std::to_string(line_number) +
                                "\n" + err.what());
     }
 
     catch (const std::logic_error& err)
     {
-      throw std::logic_error("Error reading OpenSn cross section file "
+      throw std::logic_error("Error reading Chi cross section file "
```

Comment:
github's diff shows that the old file had "OpenSn cross section" and that the new version of the file has "Chi cross section". Weird.

Reply:
That's weird. I'll double check that.